tests/journald_tests.cpp

```diff
@@ -294,9 +299,10 @@ TEST_F(JournaldLoggerTest, ROOT_LogToJournaldWithBigLabel)
 }
 
 
-// This checks a specific case where the arguments passed into the
-// ContainerLogger will differ if the Mesos agent restarts before
-// launching a nested container.
+// This test verfies that the executor information will be passed to
```

Comment:
s/verfies/verifies/

Reply:
Dang, I copied that from somewhere - need to fix the original then as well..... Thanks!